lib/parse_html.js

```diff
@@ -42,6 +42,7 @@ module.exports = function parseHTML(context) {
 function createRootNode(nodes) {
   const root  = {
     type:     'root',
+    attribs:  {},
```

Comment:
schrodinger's code. It's both correct and incorrect and you don't know until you test.